tech-lady-zz/fly-app
# #150034644 Set up database schema

#### What does this PR do?
This PR adds database schema for the application

#### Description of Task to be completed?
**AC**:
- Setup your database using sqlite3
- Run migrations to generate a schema

#### How should this be manually tested?
- Navigate to the console using `rails dbconsole`
- Check existing tables using `.tables`
- Exit from the console using `.exit`

#### What are the relevant pivotal tracker stories?
#150034644
